.docker.test.py

```diff
@@ -26,6 +26,7 @@
         'PY3',
         'SED',
         'TEXT',
+        'ZIG',
```

Comment:
aarch64 only tests tier 1, and Zig is in tier 3, so this will fail all the time.

Reply:
oh do I need to add it anywhere else?

Reply:
No, it should be picked up by `get_available` in the x86_64 branch above.